Source/WebKit/Scripts/webkit/tests/TestWithEnabledByMessageReceiver.cpp

```diff
@@ -50,27 +48,21 @@ void TestWithEnabledBy::didReceiveMessage(IPC::Connection& connection, IPC::Deco
     if (decoder.messageName() == Messages::TestWithEnabledBy::ConditionallyEnabled::name()) {
         if (sharedPreferences && sharedPreferences->someFeature)
             return IPC::handleMessage<Messages::TestWithEnabledBy::ConditionallyEnabled>(connection, decoder, this, &TestWithEnabledBy::conditionallyEnabled);
-        runtimeEnablementCheckFailed = true;
+        return decoder.markInvalid();
```

Comment:
Don't we want to add a release log here?

Reply:
Will add.